### PR TITLE
[Aviraj] Added pre-test condition

### PR DESCRIPTION
**Pull request by avirajkhare00**
### Description
I have added a pre test condition in Jest. If Aptos local node is not running with indexer. Tests will not start

### Test Plan
Run `pnpm test` with and without localnode to test it out.

### CODE REVIEW

**Review by 0xmaayan — COMMENTED** [review]
PR description says that if Indexer is not up, test wont run - but the code actually check if the node is not up.

Also, not sure `ps -ax | grep aptos` is compatible with windows - might be only in Unix-like OS.

I think a better check would be to actually ping or check whether `localhost:8080` is up and if, for example, `localhost:8080/v1` returns a response.

File: jest.config.js [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
-    global: {
-      branches: 50, // 90,
-      functions: 50, // 95,
-      lines: 50, // 95,
-      statements: 50, // 95,
-    },
-  },
-  // To help avoid exhausting all the available fds.
-  maxWorkers: 4,
+	preset: "ts-jest",
```
Comment (CapCap) [inline jest.config.js:3]: please use space indent instead of tabs, and please align your linter with existing file, thanks!

**Comment by avirajkhare00** [comment]
Hey @CapCap. Please check 🙏

**Comment by 0xmaayan** [comment]
Closing since running local testnet with indexer process while running tests was implemented in the sdk

**Pull request closed without merging**